docs/workflows/openapi-contract.md

```
@@ -67,7 +70,16 @@ Optional but recommended:
    ```
    The command fails if artifacts are stale or missing.
 
-4. **Run quality gates locally**
+4. **Verify the contract version/shape guard**
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)